samples/test/exportDataToBigquery.test.js

```diff
@@ -0,0 +1,70 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+'use strict';
+
+const {assert} = require('chai');
+const {after, before, describe, it} = require('mocha');
+const uuid = require('uuid');
+const cp = require('child_process');
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+
+const GCLOUD_TESTS_PREFIX = 'nodejs_samples_tests';
+const generateUuid = () =>
+  `${GCLOUD_TESTS_PREFIX}_${uuid.v4()}`.replace(/-/gi, '_');
+
+const {
+  ContactCenterInsightsClient,
+} = require('@google-cloud/contact-center-insights');
+const client = new ContactCenterInsightsClient();
+
+const {BigQuery} = require('@google-cloud/bigquery');
+const bigquery = new BigQuery();
+
+const bigqueryDataset = generateUuid();
+const bigqueryTable = generateUuid();
+
+describe('ExportInsightsData', () => {
+  let projectId;
+  let bigqueryProjectId;
+
+  before(async () => {
+    projectId = await client.getProjectId();
```

Comment:
Are there situations where these two projectIds would be different!?

Reply:
Yes! The user can create Contact Center Insights resources in one project and export these resources to BigQuery tables in another project that they own. This feature is documented here https://cloud.google.com/contact-center/insights/docs/export#export_data_to_another_project_optional